openid-4-verifiable-presentations-1_0.md

```diff
@@ -1611,17 +1611,30 @@ The signed request allows the Wallet to authenticate the Verifier using a trust
 
 ## Response
 
-Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The Authorization Response is a JavaScript object, where the response parameters as defined for the Response Type are encoded as top-level members in this JavaScript object. 
+Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The response is a JavaScript object as defined in [@!w3c.digital_credentials_api], where the Authorization Response parameters as defined for the Response Type are encoded as members within the `data` member of this JavaScript object.
```

Comment:
since the vp_token passes through the use-agent, it would be recommended to encrypt the JWT containing the response using the RP public key

Reply:
there is a chance encryption will be mandatory in W3C spec, so i think we are waiting for the decision there?

Reply:
There's an issue that mentions encryption here:

https://github.com/openid/OpenID4VP/issues/203

We might need a more focussed issue on it, but I would like to keep it out of scope of this PR as we need to reach consensus in the working group(s) first.